### DISCUSSION TITLE
Quien soy?

### DISCUSSION THREAD

**Discussion by Matiasprom1541** [community/community#63580] [Other Feature Feedback, Questions, & Ideas] [answered]
Soy programador se html,css javascript pero estoy estudiando basado en objetos,sass basico,flexbox estoy aprendiendo c#

**Comment by queenofcorgis** [comment, marked as answer]
👋 Welcome to the GitHub Community on Discussions, @Matiasprom1541  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.


As a reminder, we take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Reply by Matiasprom1541** [reply]
Ok gracias entiendo pero este es mi idioma

**Comment by Matiasprom1541** [comment]
Si entiendo pero hay usuarios que escrieben en español y no los
notificán...nesecito el github ya que estoy trabajando en proyectos haci
que podria subir en privado??ya que no se ingles muy bien para ver su
mensaje tube que usar traductor...........

**Reply by Scorhenao** [reply]
deja de perter el tiempo mejor ponte a estudiar ingles XD
